package/src/mock-builders/api/queryMembers.js

```diff
@@ -14,3 +14,101 @@ export const queryMembersApi = (members = []) => {
 
   return mockedApiResponse(result, 'get');
 };
+
+export const CHANNEL_WITH_ONE_MEMBER_MOCK = {
+  okey: {
+    banned: false,
+    channel_role: 'channel_member',
+    created_at: '2021-01-27T11:54:34.173125Z',
+    role: 'member',
+    shadow_banned: false,
+    updated_at: '2021-02-12T12:12:35.862282Z',
+    user: {
+      id: 'okechukwu nwagba martin',
+      name: 'okechukwu nwagba martin',
+    },
+    user_id: 'okechukwu nwagba martin',
+  },
+};
+
+export const GROUP_CHANNEL_MOCK = {
```

Comment:
These are mocked members and not a channel - could it be nice to name it as such?